src/collections/blog/2024/09-27-meet-the-maintainer-aisuko-li/post.mdx

```diff
@@ -0,0 +1,298 @@
+    I used to work for RancherLabs for a while, and I worked to maintain the official Helm (a third-party management tool for Kubernetes manifests) charts repo. These experiences helped me contribute to creating and maintaining Meshery Helm charts.
+    <br />
+    I like open source software, and I love contributing to the community.
+    The more you contribute, the more permission you get to help the
+    community grow and improve.
+    <br />
+    The Layer5 community is a true open source community. Everyone here can find
+    a comfortable role. I have been here since 2019 (a long time ago). I’ve seen
+    new members join and some leave. It's great to see people work together without
+    any other conditions. This is one of the ways I have fun.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Anita:</span>
+  <p>
+    You’ve been consistently contributing to a large number of Layer5 projects
+    (Meshery adapters, mesheryctl, SMI, SMP). Layer5 has a large collection of
+    active projects. Which one are you currently focusing on? <i>Psst.</i> Also,
+    which one’s your favorite? I won’t tell.
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aisuko:</span>
+  <p>
+    Actually, the Meshery project in 2021-2022 has changed a lot. More skilled and
+    talented contributors joined the community. They are so professional and
+    active, and their hard work has made Meshery more powerful than before. For
+    instance, projects like `meshkit` and `meshsync` have grown significantly.
+    It’s great to have such a strong team working together.
+    <br />
+    Right now, I am primarily focusing on the `meshery-operator` project and
+    `meshery-linkerd`, along with fixing bugs across all the projects. I always
+    aim to make all the projects more controllable and maintain high code quality.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Anita:</span>
+  <p>
+    Have you worked with any other open source projects? How does Layer5 compare?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aisuko:</span>
+  <p>
+    I was active in the Rancher community and the Helm charts project, where I owned
+    three charts. I’m also still a maintainer of the GNU Hurd. Recently, I’ve been
+    working on contributions to Kubernetes community projects as well.
+    <br />
+    Compared to the Layer5 community, the Kubernetes community is much larger. 
+    Many members are not very active, so it can be difficult to get feedback 
+    on PRs and issues from inactive members.
+    <br />
+    The GNU Hurd project is unique, so there’s no need to compare it with others.
+    In the Layer5 community, we have a warm welcome for new contributors, and most
+    projects have active reviewers who provide feedback quickly.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Anita:</span>
+  <p>What is so fascinating about service meshes?</p>
```

Comment:
@hargunkaur286 terrible question. I've repeatedly asked that we distance the project from service meshes as much as possible. Will you please see that this question is either removed or replaced with another?